pkg/integration/tests/custom_commands/basic_at_runtime.go

```diff
@@ -0,0 +1,57 @@
+package custom_commands
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var BasicAtRuntime = NewIntegrationTest(NewIntegrationTestArgs{
```

Comment:
I would have two tests: one for the basic case of just adding a file, and one for the case of storing history. This test is combining the two and makes it a little harder to follow.

Reply:
> one for the basic case of just adding a file, and one for the case of storing history

Hmmm just to confirm, one test for confirming the command _is_ saved, and another for confirming that it _isn't_ saved?

Reply:
one test to confirm that the command is executed (e.g. creating a file) and a separate test to test the history logic (so you could rename the current test to be about history and make a separate smaller test that just asserts the command is executed)